tox.ini

```diff
@@ -65,6 +68,7 @@ deps =
   flake8~=3.7
   isort~=4.3
   black>=19.3b0,==19.*
+  opentracing~=2.2
```

Comment:
This is a runtime dependency so it should be specified in the shim's setup.py and not be required here.

Reply:
Thanks. I've fixed `setup.py`. in the meantime. Removed from `tox.ini`.